pkg/cmd/output.go

```diff
@@ -30,14 +29,14 @@ func formatFloat(f float64) string {
 	return fmt.Sprintf("%.6f", f)
 }
 
-func writeAllocationTable(out io.Writer, allocationType string, allocations map[string]kubecost.Allocation, opts displayOptions, currencyCode string, showNamespace bool) {
-	t := makeAllocationTable(allocationType, allocations, opts, currencyCode, showNamespace)
+func writeAllocationTable(out io.Writer, allocationType string, allocations map[string]kubecost.Allocation, opts displayOptions, currencyCode string, showNamespace bool, isHistorical bool) {
```

Comment:
I'd recommend calling this new arg `projectToMonthlyRate` or something more along those lines.

Reply:
That keeps this function a little more abstract, though it will force you to just flip the truth value in all calls.